src/apps/TCO2_Dashboard/app.py

```diff
@@ -0,0 +1,311 @@
+                     methodology_volume, pool_pie_chart, eligible_pool_pie_chart
+from figures_carbon_pool import deposited_over_time, redeemed_over_time
+from tco2 import create_content_toucan
+from bct import create_content_bct
+from helpers import date_manipulations, region_manipulations, subsets, drop_duplicates, filter_carbon_pool
+from data_related_constants import rename_map, retires_rename_map, deposits_rename_map, redeems_rename_map
+from subgrounds.subgrounds import Subgrounds
+from flask_caching import Cache
+
+
+app = dash.Dash(title="KlimaDAO Tokenized Carbon Dashboard", suppress_callback_exceptions=True,
+                external_stylesheets=[dbc.themes.BOOTSTRAP])
+cache = Cache(app.server, config={
+    'CACHE_TYPE': 'filesystem',
+    'CACHE_DIR': 'cache-directory'
+})
+TIMEOUT = 86400
+
+
+@cache.memoize(timeout=TIMEOUT)
+def get_data():
+
+    sg = Subgrounds()
+    carbon_data = sg.load_subgraph(
+        'https://api.thegraph.com/subgraphs/name/cujowolf/polygon-bridged-carbon')
+
+    carbon_offsets = carbon_data.Query.carbonOffsets(
+        orderBy=carbon_data.CarbonOffset.lastUpdate,
+        orderDirection='desc',
+        first=999
+    )
+
+    df_bridged = sg.query_df([
+        carbon_offsets.tokenAddress,
+        carbon_offsets.region,
+        carbon_offsets.vintage,
+        carbon_offsets.projectID,
+        carbon_offsets.standard,
+        carbon_offsets.methodology,
+        carbon_offsets.balanceBCT,
+        carbon_offsets.balanceNCT,
+        carbon_offsets.totalBridged,
+        carbon_offsets.bridges.value,
+        carbon_offsets.bridges.timestamp
+    ])
+
+    carbon_offsets = carbon_data.Query.retires(
+        first=999
+    )
+
+    df_retired = sg.query_df([
+        carbon_offsets.value,
+        carbon_offsets.timestamp,
+        carbon_offsets.offset.tokenAddress,
+        carbon_offsets.offset.region,
+        carbon_offsets.offset.vintage,
+        carbon_offsets.offset.projectID,
+        carbon_offsets.offset.standard,
+        carbon_offsets.offset.methodology,
+        carbon_offsets.offset.standard,
+        carbon_offsets.offset.totalRetired,
+    ])
+
+    return df_bridged, df_retired
+
+
+@cache.memoize(timeout=TIMEOUT)
+def get_data_pool():
+
+    sg = Subgrounds()
+    carbon_data = sg.load_subgraph(
+        'https://api.thegraph.com/subgraphs/name/cujowolf/polygon-bridged-carbon')
+
+    carbon_offsets = carbon_data.Query.deposits(
+        first=999
+    )
+
+    df_deposited = sg.query_df([
+        carbon_offsets.value,
+        carbon_offsets.timestamp,
+        carbon_offsets.pool,
+        # carbon_offsets.offset.region,
+    ])
+
+    carbon_offsets = carbon_data.Query.redeems(
+        first=999
+    )
+
+    df_redeemed = sg.query_df([
+        carbon_offsets.value,
+        carbon_offsets.timestamp,
+        carbon_offsets.pool,
+        # carbon_offsets.offset.region
+    ])
+
+    return df_deposited, df_redeemed
+
+
+df, df_retired = get_data()
```

Comment:
Hmm I don't think this setup will actually refetch the data and update the charts when the cache expires 

We need to wrap the entire definition of `app.layout` into a function like they do in this example: https://dash.plotly.com/live-updates#updates-on-page-load

Then you just make that layout function cached instead of having to cache the `get_data` and `get_data_pool` functions and call them again